### PR TITLE
Updated for full API hiding

### PR DESCRIPTION
**Pull request by pamgoodrich**
Updated visibility doc to include full API.

### CODE REVIEW

File: 4.-documentation/set-internal-docs.md [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
-- An article contains internal notes that you want visible to workspace members (except for guests), but not to public users. This is useful for linking to other articles and API elements in the same project without exposing your internal notes.
+- An API operation on an otherwise publicly available API is only intended for internal use. In this case, set the operation as internal.
+- A project or a repository contains a mix of public and internal APIs and you want to keep the documentation together in the same project. In this case, set the internal flag for APIs you want to hide.
+- An article is in draft form. This is especially useful for web projects because it allows you to stage articles before making them public. In this case, set the internal flag for the Markdown file.
+- An article contains internal notes that you want visible to workspace members (except for guests), but not to public users. This is useful for linking to other articles and API elements in the same project without exposing your internal notes. In this case, set the internal flag for the Markdown file.
```
Comment (kaylachun) [inline 4.-documentation/set-internal-docs.md:13]: Looks good! Maybe adding something in like:
"Setting the entire API as internal will also make all of its endpoints and models internal.  However, if you choose to set the API as public again, the endpoints and model's previous internal settings are preserved."
Basically, setting the API as internal does not remove any of the previous internal endpoint's state.. just overrides it until the API is public again.

**Review by kaylachun — COMMENTED** [review]
Looks great! Just added a comment about the saved internal state.

File: 4.-documentation/set-internal-docs.md (hunk 1 above)
Comment (pamgoodrich) [reply 4.-documentation/set-internal-docs.md:13]: Oh! That's good to know! Thank you!

**Review by kaylachun — APPROVED** [review]
Looks good!!

File: 4.-documentation/set-internal-docs.md [hunk 2]
```diff
@@ -1,34 +1,43 @@
 # Set Article and API Visibility
 
-You can prevent articles, paths, and models from being visible to users who are not members of a workspace and guest users by marking those items as internal.
+You can hide API, paths, models, and articles from public documentation and guest users by marking those items as internal.
```
Comment (heitortsergent) [inline 4.-documentation/set-internal-docs.md:3]: API -> APIs to match other items?

**Review by heitortsergent — APPROVED** [review]
Looks great, just had one small comments!

**Pull request merged**